### PR TITLE
Updated quantize_weights.cc

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Fixed warnings in the file

### CODE REVIEW

**Review by srjoglekar246 — APPROVED** [review]

File: tensorflow/lite/tools/optimize/quantize_weights.cc [hunk 1]
```diff
@@ -52,12 +52,12 @@ std::vector<ConsumerOpInfo> GetTensorConsumers(const ModelT* model,
...
-  for (int op_idx = 0; op_idx < subgraph->operators.size(); ++op_idx) {
+  for (size_t op_idx = 0; op_idx < subgraph->operators.size(); ++op_idx) {
     OperatorT* op = subgraph->operators[op_idx].get();
     if (op == nullptr) {
       continue;
     }
-    for (int i = 0; i < op->inputs.size(); ++i) {
+    for (size_t i = 0; i < op->inputs.size(); ++i) {
       if (op->inputs[i] == tensor_idx) {
         consumer_ops.push_back({op, op_idx, i});
```
Comment (srjoglekar246) [inline tensorflow/lite/tools/optimize/quantize_weights.cc]: We need to `static_cast<int>` `op_idx` & `i` here, since the initializer expects integers.

**Review by srjoglekar246 — CHANGES_REQUESTED** [review]

File: tensorflow/lite/tools/optimize/quantize_weights.cc (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/tools/optimize/quantize_weights.cc]: @srjoglekar246 , thanks for your review comments, i have updated the code, kindly check and approve.

**Comment by amitsrivastava78** [comment]
@srjoglekar246 , thanks for your comments, i have updated the code as per the comments, kindly check and approve.

File: tensorflow/lite/tools/optimize/quantize_weights.cc [hunk 2]
```diff
@@ -52,14 +52,14 @@ std::vector<ConsumerOpInfo> GetTensorConsumers(const ModelT* model,
...
+  for (size_t op_idx = 0; op_idx < subgraph->operators.size(); ++op_idx) {
     OperatorT* op = subgraph->operators[op_idx].get();
     if (op == nullptr) {
       continue;
     }
-    for (int i = 0; i < op->inputs.size(); ++i) {
+    for (size_t i = 0; i < op->inputs.size(); ++i) {
       if (op->inputs[i] == tensor_idx) {
-        consumer_ops.push_back({op, op_idx, i});
+        consumer_ops.push_back({op, static_cast<int> (op_idx), static_cast<int> (i)});
```
Comment (srjoglekar246) [inline tensorflow/lite/tools/optimize/quantize_weights.cc]: Style change:

```
consumer_ops.push_back(
    {op, static_cast<int>(op_idx), static_cast<int>(i)});
```

**Review by srjoglekar246 — CHANGES_REQUESTED** [review]

File: tensorflow/lite/tools/optimize/quantize_weights.cc (hunk 2 above)
Comment (amitsrivastava78) [reply tensorflow/lite/tools/optimize/quantize_weights.cc]: @srjoglekar246 , thanks for the comments, i have updated the same, kindly check and approve,

**Comment by amitsrivastava78** [comment]
@srjoglekar246 , i have updated the code as per your comments, kindly check and approve.

**Review by srjoglekar246 — APPROVED** [review]

**Pull request merged**